receiver/windowsperfcountersreceiver/README.md

```diff
@@ -25,11 +22,25 @@ be configured:
 ```yaml
 windowsperfcounters:
   collection_interval: <duration> # default = "1m"
+  metric_metadata:
```

Comment:
I think the examples here need a once over to pick up the updates you've made in the actual code.

Reply:
Whoops, yup went through and cleaned them up.